Renovate bot should use project's gradle wrapper

**Describe the bug**
`"gradle --init-script renovate-plugin.gradle renovate"` is executed with your own gradle instance (right now it's using a gradle with version `5.6.4`). However, it should use the project wrapper to init this plugin (or al least the same version).

For example, [Gradle Play publisher min gradle required version is](https://github.com/Triple-T/gradle-play-publisher/commit/a6473a5eaa10171849ef13724a47bd3e0df74656) is 6.2.
So if you have this plugin integrated, renovate bot will not work. Same issue if you use [android-snapshot-publisher plugin](https://github.com/xmartlabs/android-snapshot-publisher)

**Did you see anything helpful in debug logs?**
```
{
  "errMessage": "Command failed: docker run --rm -v \"/mnt/renovate/gh/**************\" -v \"/tmp/renovate-cache\":\"/tmp/renovate-cache\" -w \"/mnt/renovate/gh/*******\" renovate/gradle bash -l -c \"gradle --init-script renovate-plugin.gradle renovate\"\n\nFAILURE: Build completed with 2 failures.\n\n1: Task failed with an exception.\n-----------\n* Where:\nBuild file '/mnt/renovate/***********/app/build.gradle' line: 18\n\n* What went wrong:\nA problem occurred evaluating project ':app'.\n> Failed to apply plugin [id 'com.xmartlabs.snapshot-publisher']\n   > Android Snapshot Publisher's minimum Gradle version is at least Gradle 6.0 and yours is Gradle 5.6.4. Find the latest version at https://github.com/gradle/gradle/releases, then run './gradlew wrapper --gradle-version=$LATEST --distribution-type=ALL'.\n\n* Try:\nRun with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.\n==============================================================================\n\n2: Task failed with an exception.\n-----------\n* What went wrong:\nA problem occurred configuring project ':app'.\n> compileSdkVersion is not specified.\n\n* Try:\nRun with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.\n==============================================================================\n\n* Get more help at https://help.gradle.org\n\nBUILD FAILED in 3m 46s\n"
}
```
**To Reproduce**
Create a project and integrate [android-snapshot-publisher plugin](https://github.com/xmartlabs/android-snapshot-publisher) or [gradle-play-publisher](https://github.com/Triple-T/gradle-play-publisher)

--- a/lib/manager/gradle/index.ts
+++ b/lib/manager/gradle/index.ts
@@ -24,7 +24,6 @@ import {
 import { platform } from '../../platform';
 import { LANGUAGE_JAVA } from '../../constants/languages';
 import { DATASOURCE_MAVEN } from '../../constants/data-binary-source';
-import { BinarySource } from '../../util/exec/common';
 import { DatasourceError } from '../../datasource';
 
 export const GRADLE_DEPENDENCY_REPORT_OPTIONS =
@@ -45,8 +44,6 @@ async function getGradleCommandLine(
   cwd: string
 ): Promise<string> {
   const args = GRADLE_DEPENDENCY_REPORT_OPTIONS;
-
-  if (config.binarySource === BinarySource.Docker) return `gradle ${args}`;
 
   const gradlewPath = upath.join(cwd, 'gradlew');
   const gradlewExists = await exists(gradlewPath);

